src/Cosi-BLS/new-transactions.lisp

```diff
@@ -554,7 +554,8 @@ OBJECTS. Arg TYPE is implicitly quoted (not evaluated)."
                             (- sum-of-inputs sum-of-outputs)
                             *minimum-transaction-fee*)
                       (return nil)))
-                  (format t "~%Successful transaction ~a~%  TxID = ~a~%" 
+                  (format t "~%Node ~a declares Successful transaction ~a~%  TxID = ~a~%"
+                          (cosi-simgen::current-node)
```

Comment:
The double-colon should not be used to reference cosi-simgen:current-node, which is an exported symbol.